cmake/Modules/OpmLibMain.cmake

```diff
@@ -19,7 +19,10 @@
 #
 #   - CMP0026 to allow access to the LOCATION target property
 #   - CMP0048 to indicate that we want to deal with the *VERSION*
-#     variables ourselves
+#							variables ourselves
```

Comment:
There seems to be some tab-vs-space going on here.

I seem to remember that in the old (autotools) build system there were some things that should be tabs and not spaces for some technical reason, and that is why we had tabs in build system files (I may be mistaken about this). Anyway, we should probably get rid of tabs in cmake files, since there is no technical reason to keep them.